src/bridge.ts

```diff
@@ -0,0 +1,69 @@
+import { BigInt, dataSource } from '@graphprotocol/graph-ts';
+import { CCIPSendRequested } from '../generated/CCIPOnRamp/CCIPOnRamp';
+import { BridgeRequestTransaction, Position } from '../generated/schema';
+import { config } from './config';
+
+export function handleCCIPSendRequested(event: CCIPSendRequested): void {
+  const networkConfig = config.get(dataSource.network());
+
+  const transactionHash = event.transaction.hash.toHexString();
+  const receiver = event.params.message.receiver.toHexString();
+
+  const bridgeRequestTransaction = loadOrCreateBridgeRequestTransaction(transactionHash);
+  const position = loadOrCreatePosition(receiver);
+
+  bridgeRequestTransaction.position = position.id;
+
+  if (event.params.message.tokenAmounts.length > 0) {
+    const token = event.params.message.tokenAmounts.at(0).token.toHexString();
+
+    let rTokenAddress = '';
+    if (networkConfig != null) {
+      const address = networkConfig.get('R');
+      rTokenAddress = address !== null ? (address as string) : '';
+    }
+    // Skip bridge transaction if bridged token is not R
+    if (token != rTokenAddress) {
+      return;
+    }
+
+    bridgeRequestTransaction.amount = event.params.message.tokenAmounts.at(0).amount;
+  } else {
+    // Skip bridge transaction if token data is not present
+    return;
+  }
+
+  let destinationChainSelector = '';
+  if (networkConfig != null) {
```

Comment:
```suggestion
  if (networkConfig !== null) {
```

Reply:
Code gets converted to WebAssembly, and !== doesn't work as expected.